tests/test_D2_model_constraints.py

```diff
@@ -262,9 +265,14 @@ def test_add_constraints_minimal_renewable_share_None(self):
                 }
             }
         )
-        # todo to be added
-        pass
-    """
+        model = D2.add_constraints(
+            local_energy_system=self.model,
+            dict_values=self.dict_values,
```

Comment:
```suggestion
            dict_values=dict_values,
```
Here you should use the adapted dict_values; set renewable energy share in the line above to 0. `MINIMAL_RENEWABLE_FACTOR: {VALUE: 0}`